sdk/translation/Azure.AI.Translation.Document/src/BatchDocumentTranslationModelFactory.cs

```diff
@@ -0,0 +1,159 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using Azure.Core;
+
+namespace Azure.AI.Translation.Document.Models
```

Comment:
As described in issue https://github.com/Azure/azure-sdk-for-net/issues/22659, we will like to move this class to the main namespace `Azure.AI.Translation.Document`.
We will also need to rename the class to `DocumentTranslationModelFactory`